ams/server/src/main/java/com/netease/arctic/server/IcebergRestCatalogService.java

```diff
@@ -137,10 +137,11 @@ public void handleException(Exception e, Context ctx) {
     jsonResponse(ctx, response);
     if (code.code >= 500) {
       LOG.warn("InternalServer Error", e);
+    } else {
```

Comment:
Why do we need to reduce the logging level for errors in which the code is smaller than 500 here?

Reply:
because those errors happened when the client-side passed arguments with problems. 
The service side usually does not need to log those errors.

Reply:
Got it.
Maybe add some comments here to tell that.